### ISSUE TITLE
Is it possible to add attributes to the htmx script tag

### ISSUE THREAD

**Issue by mikeymeister** [putyourlightson/craft-sprig#338]
Prior to the depreciation of `{{ sprig.script }}`, we made use of the option to pass in a hash of tag attributes. In one case to make sure cookie management software knew to never block the loading of that script eg. `{{ sprig.script({'data-cookieconsent':'ignore'}) }}`.

I can't see anyway to do this now, am I missing something or has this functionality been removed with the depreciation of `sprig.script` ?

Thanks

**Comment by bencroker** [comment]
You’re right that this functionality was lost. I’ve added the ability to pass script tag attributes via the `sprig.registerScript()` and `sprig.setRegisterScript()` functions in https://github.com/putyourlightson/craft-sprig-core/commit/c76cdd47e05bb9d7804c72ca9425cc73aaa56d0d. The latter should do the trick for you when added to the top of the layout template.

```twig
{% do sprig.setRegisterScript({ 'data-cookieconsent': 'ignore' }) %}
```

You can test this by running `composer require "putyourlightson/craft-sprig-core:dev-develop as 2.6.4"`.

**Comment by bencroker** [comment]
Released in Sprig [2.7.3](https://github.com/putyourlightson/craft-sprig/releases/tag/2.7.3).
